### ISSUE TITLE
fix context-menu

### ISSUE THREAD

**Issue by skalimer0** [mrliptontea/PurpleMine2#246]
Hello,

On Firefox the context-menu is not attach on mouse position.

You fix this on SidebarToogle.js :

`    
// Fix issue with context menu position
    if (this.$main.css('position') === 'relative') {
      $(window).on('load', function () {
        $('#context-menu').appendTo('#wrapper3')
      })
    }
`

It's OK on Chrome, but i think for Firefox it need to be on 'ready' event not 'load' or add detach() method before appendTo() :
`    
        $('#context-menu').detach().appendTo('#wrapper3')
`

Thanks
